frame/support/src/storage/types/counted_map.rs

```diff
@@ -412,6 +412,23 @@ where
 			res
 		})
 	}
+
+	/// Enumerate all elements in the counted map after a specified `starting_raw_key` in no
+	/// particular order.
+	///
+	/// If you alter the map while doing this, you'll get undefined results.
+	pub fn iter_from(
+		starting_raw_key: Vec<u8>,
```

Comment:
I just noticed this, but we really can just use `&[u8]` here instead of mandating a `Vec` here to allow for a more flexible API.

Reply:
If you want to have a flexible api, you should use `impl Into<Vec<u8>>`

Reply:
What's the benefit compared to `&[]`?

Reply:
all in all though @KiChjang I think if we change the API here, we should do so as well for many other exposed functions as well in the storage API, so I'd rather do it in a separate PR. For now, I prefer staying consistent with the rest of the API.

Reply:
these are zero cost abstractions right? in which case, yeah let's follow up on this